test/test_nodes.jl

```diff
@@ -275,7 +275,7 @@ end
 
         resources = [NG, Power, CO2]
         ops = SimpleTimes(5, 2)
-        op_per_strat = duration(ops)
+        op_per_strat = TS._total_duration(ops)
```

Comment:
`_total_duration` not part of the TS API, PR to change that or is there an alternative?

Reply:
In practice, I used `_total_duration` for laziness and fast potential to switch the operational periods, while keeping the overall calculations the same. This required unfortunately a few more modifications to avoid issues with it, but it is by no means a requirement..